src/main/webapp/app/exercises/quiz/shared/questions/quiz-scoring-infostudent-modal/quiz-scoring-info-student-modal.component.ts

```diff
@@ -46,7 +46,6 @@ export class QuizScoringInfoStudentModalComponent implements AfterViewInit {
     checkForWrongAnswers = new Array<AnswerOption>();
 
     /* Drag and Drop Counting Variables*/
-    wronglyMappedDragAndDropItems: number; // Amount of wrongly mapped drag and drop item
     differenceDragAndDrop: number; // Difference between the wronglyMappedDragAndDropItems and correctlyMappedDragAndDropItems
```

Comment:
it should be Difference between the **incorrectly**MappedDragAndDropItems and correctlyMappedDragAndDropItems right?